lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -345,9 +345,10 @@ <h3 class="course-title" id="course-title-${enrollment.course_id}">
       % if related_programs:
         <div class="message message-related-programs is-shown">
           <span class="related-programs-preface" tabindex="0">${_('Related Programs')}:</span>
-          <ul>
+          <ul class="related-program-container" >
             % for program in related_programs:
-            <li>
+              ## intentionally using multiple classnames here to bump up specificity
+            <li class="related-program-element buffered-element important-element">
```

Comment:
I see you've said that using multiple classnames here is intentional, but is it necessary?

Are these classnames used elsewhere? Are we being specific to be defensive?
If you've introduced these three classnames and they aren't being used elsewhere, I have a feeling that just one will do the trick.

Reply:
There were 4 additional classes driving the `<li/>` tag elsewhere. I added one classname and it didn't make a difference. I can play with it to get the least number of extra classes, but it looked like I needed to overload the number before the styling will be applied.
I can clear up that errant space!

Reply:
@ciduarte I experimented with this, and took out one classname, and refactored the li tags a bit. Will this do?